.travis.yml

```diff
@@ -60,7 +60,7 @@ before_install:
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then bash scripts/package-osx.sh; fi
 script:
 - py.test -v --cov=picard --cov-report xml:coverage.xml
-- isort --recursive --check-only .
+- isort --recursive --check-only $(git ls-tree -r --name-only $(git rev-parse HEAD) | grep "\\.py$")
```

Comment:
The check is failing on https://travis-ci.org/metabrainz/picard/builds/515581198
It is supposed  to pass on this PR, but log shows:
> ERROR: /home/travis/build/metabrainz/picard/picard/resources.py Imports are incorrectly sorted.

This file (and few others) should be skipped according to .isort.cfg configuration, may be this command doesn't find `.isort.cfg`, `--settings-path` could help perhaps.